spec/src/main/asciidoc/related-documents.adoc

```diff
@@ -5,11 +5,10 @@
 [bibliography]
 == Related Documents
 
-- [[[a19493,1]]] Enterprise JavaBeans, v. 3.0. Java Persistence API.
+- [[[a19493,1]]] Jakarta Persistence, v. 2.3.
```

Comment:
v 2.3?

Reply:
more haste less speed.... 2.2 of course. Thanks for catching this! fixed.